src/pmod_gps.erl

```diff
@@ -40,36 +53,45 @@ start_link(Slot, _Opts) ->
 
 % @doc Get the GPS data.
 %
-% The input parameter specifies which sentence to get. For a description of
-% the sentences see the
+% The input parameter specifies which type of sentence to get.
+% For a description of the sentences see the
 % <a href="https://reference.digilentinc.com/_media/reference/pmod/pmodgps/pmodgps_rm.pdf">
 % PmodGPS Reference Manual
+% The sentence CRC is checked for all the sentence types, but for now only the
+% GGA values are parsed. If other sentences are needed, grisp_nmea needs to be
+% extended to support more types.
 % </a>.
 %
 % === Example ===
 % ```
 %  2> pmod_gps:get(gga).
-%  <<"$GPGGA,145832.000,5207.3597,N,01135.6957,E,1,5,2.50,61.9,M,46.7,M,,*6F\n">>
+%  {gps,gga,#{fixed => false,alt => undefined,lat => undefined,long => undefined,time => 26799}}
```

Comment:
Should the latitude and longitude really be `undefined` in this example?

Reply:
I can't get a fix here so I couldn't get an example

Reply:
I could probably parse the previous example and use that...

Reply:
I pushed the change